### ISSUE TITLE
Typescript example in docs generates warnings in Viz

### ISSUE THREAD

**Issue by deinspanjer** [davidkpiano/xstate#246]
## Bug or feature request?
Bug

## Description:
The state machine documented [here](https://xstate.js.org/docs/guides/typescript/) has the following line toward the bottom:
`            '': { target: 'green' }`
If you take that machine configuration and paste it into the xstate-viz, the console spews lots of the following warnings:
```
Target 'green' not found on 'light.red.stop'
(anonymous) @ graph.js:118
...
n.render | @ | StateChartNode.js:133
```

## (Bug) Expected result:
To be able to explore the documentation example with Viz

## (Bug) Actual result:
It sorta works, but the red state can't hit the stop nested state and every transition gives that warning in the console

## (Bug) Potential fix:
I didn't see anything in the docs about what an empty property name would indicate, so maybe that is a typo in the example and it should actually be PED_COUNTDOWN?
Changing that doesn't fix the error though.  It seems the target should be something other than 'green'.

## Link to reproduction or proof-of-concept:
1. Visit [VIZ](https://statecharts.github.io/xstate-viz/)
2. Open the browser's console
3. Paste in the StateMachine configuration from the [typescript](https://xstate.js.org/docs/guides/typescript/) page

**Comment by davidkpiano** [comment]
Ah, that's a mistake in the documentation. It should be `'#light.green'`.

**Comment by deinspanjer** [comment]
Cool.  Maybe while in there, you could see if the example could be made a little more consistent/functional? Currently, it can't get from the wait to the stop state, and a power outage event sets the pedestrian state to walk which could be a bit dangerous if this was anything other than a hypothetical example. ;)

**Comment by deinspanjer** [comment]
Oh, regarding the can't get to stop state, I understand a little better what is happening there.  The condition prevents Viz from letting the user click on the event.
I tried manually sending the event `{type: "PED_COUNTDOWN", duration: 0}`, but that still doesn't work because `elapsed` is still 0 and I don't see a way to change that within the simulation.  I did try changing it in the definition of the machine and then manually sending the event did transition.
